<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RouteWidget</class>
 <widget class="QWidget" name="RouteWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Domain Strategy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="domainStrategyCombo">
       <item>
        <property name="text">
         <string>AsIs</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>IPIfNonMatch</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>IPOnDemand</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,1,1" columnstretch="0,1,1,1">
     <item row="0" column="3">
      <widget class="QLabel" name="label_82">
       <property name="text">
        <string>Block</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <layout class="QGridLayout" name="blockTxtLayout"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_80">
       <property name="text">
        <string>Direct</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Domain</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QGridLayout" name="directTxtLayout"/>
     </item>
     <item row="1" column="2">
      <layout class="QGridLayout" name="proxyIPLayout"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_81">
       <property name="text">
        <string>Proxy</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <layout class="QGridLayout" name="proxyTxtLayout"/>
     </item>
     <item row="1" column="3">
      <layout class="QGridLayout" name="blockIPLayout"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>IP</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QGridLayout" name="directIPLayout"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="importRulesBtn">
       <property name="text">
        <string>Import Rules</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportRulesBtn">
       <property name="text">
        <string>Export Rules</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
